test/CMakeLists.txt

```
@@ -165,7 +170,7 @@ if(NRN_ENABLE_PYTHON AND PYTEST_FOUND)
       find_python_module(mpi4py)
       if(mpi4py_FOUND)
         add_test(
-                NAME coreneuron_mpi_spikes_py  COMMAND ${MPIEXEC} -np 2 ${PYTHON_EXECUTABLE}
+                NAME coreneuron_mpi_spikes_py  COMMAND ${MPIEXEC} -np 2  ${PYTHON_EXECUTABLE}
```

nitpick / accidentally added this?
```suggestion
                NAME coreneuron_mpi_spikes_py  COMMAND ${MPIEXEC} -np 2 ${PYTHON_EXECUTABLE}
```

---

Yup. Let's be green and tackle it with a cmake-format PR :)